google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta2/JsonToProtoMessageTest.java

```diff
@@ -345,8 +391,32 @@ public void testAllTypes() throws Exception {
     }
   }
 
+  @Test
+  public void testRepeatedTypeBytesWithLimits() throws Exception {
+    for (Map.Entry<Descriptor, String> entry : ByteRepeatedTypesToDebugMessageTest.entrySet()) {
+      int success = 0;
+      for (JSONObject json : simpleJSONAArrayBytes) {
+        try {
+          DynamicMessage protoMsg =
+              JsonToProtoMessage.convertJsonToProtoMessage(entry.getKey(), json);
+          assertEquals(protoMsg, BytesRepeatedCorrectProto.get(entry.getKey())[success]);
+          success += 1;
+        } catch (IllegalArgumentException e) {
+          assertEquals(
+              "Error: root.test_repeated[0] could not be converted to byte[].", e.getMessage());
+        }
+      }
+      if (entry.getKey() == RepeatedBytes.getDescriptor()) {
+        assertEquals(1, success);
+      } else {
+        assertEquals(0, success);
+      }
+    }
+  }
+
   @Test
   public void testAllRepeatedTypesWithLimits() throws Exception {
+    // Mark1
```

Comment:
?

Reply:
Oops, that was just so I could ctrl+f around the code easier